### PR TITLE
Support fibmatch flag for RouteGetWithOptions

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
Support `ip route get fibmatch` equivalent with RouteGetWithOptions by adding FIBMatch flag to RouteGetOptions.

### CODE REVIEW

File: route_test.go [hunk 1]
```diff
@@ -2019,3 +2019,70 @@ func TestRouteFWMarkOption(t *testing.T) {
...
+		t.Fatal(err)
+	}
+
+	// if err := RouteAdd(&Route{
+	// 	LinkIndex: link.Attrs().Index,
+	// 	Gw:        net.IPv4(192, 168, 0, 1),
+	// 	Dst:       nil,
+	// }); err != nil {
+	// 	t.Fatal(err)
+	// }
```
Comment (tklauser) [inline route_test.go:2054]: Is this still needed? Otherwise it's probably better to delete the code.

Comment (YutaroHayakawa) [reply route_test.go]: Woops, I just forgot to delete this. Fixed in the latest push.

**Comment by tklauser** [comment]
@aboch @vishvananda could you please take a look?

**Comment by aboch** [comment]
LGTM

**Pull request merged**